Dockerfile

```diff
@@ -2,25 +2,21 @@ FROM node:14-alpine
 ENV IS_DOCKER true
 WORKDIR /app
 
-# Install system packages
 RUN apk add --no-cache --no-progress \
     ca-certificates \
     python3 \
     git \
     tzdata
 
-# Upgrade pip and install Pipenv
```

Comment:
Please keep the comment.

Reply:
Same as `apk add`, it is obvious.

Do you see any case where this is useful ?

Sure, I could add it again, if you really want to.